.github/workflows/ci.yml

```diff
@@ -365,6 +365,14 @@ jobs:
             ubuntu-latest-nightly-
             ubuntu-latest-
 
+      - name: Install dashboard frontend
```

Comment:
Why was it working without this ?

Reply:
I can't remember what this fixed at the time, but it might have been fixed by other changes since. I can try to remove it and see if CI still works?

Reply:
You should be able to try to reproduce locally before...

Reply:
:facepalm:

It doesn't work without first getting the submodule and building the dashboard, and fails at building the `Asset` type (`/bee-plugin-dashboard/src/asset.rs`). I assume this must have worked before because the `dashboard` feature not being enabled in the doc command must have caused this to skip compilation. I'm not certain why it doesn't still do that that though, since `bee-plugin-dashboard` is an optional dependency....